apps/client/.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@metehancelik  you can also add a step for type checking here if you like. Personally I use it to guarantee type safety before committing.